src/eth/Web3Service.js

```diff
@@ -224,6 +224,10 @@ export default class Web3Service extends PrivateService {
   }
 
   _isStillConnected() {
+    // only determine network change as disconnect if service is connected
+    if (!this.manager().isConnected()) {
```

Comment:
Avoids an undefined on the getId error if function is called before service is connected